base/fs.cpp

```diff
@@ -194,7 +205,8 @@ std::string get_relative_path(const std::string& filename, const std::string& ba
   auto itFrom = baseDirs.begin();
   auto itTo = toParts.begin();
 
-  while (itFrom != baseDirs.end() && itTo != toParts.end() && *itFrom == *itTo) {
+  while (itFrom != baseDirs.end() && itTo != toParts.end() &&
+         base::string_to_lower(*itFrom) == base::string_to_lower((*itTo))) {
```

Comment:
We cannot do this on Linux (and sometimes depending on the configuration of the macOS file system), but I'd be worried only for Linux which the file system is case sensitive.